uaclient/apt_news.py

```diff
@@ -7,6 +7,7 @@
 
 import apt_pkg
 
+import uaclient.files.machine_token as machine_token
```

Comment:
any particular reason for the different import style here?
```suggestion
from uaclient.files import machine_token
```

Reply:
Just noticed this is used throughout - is there a difference between the two?

Reply:
Honestly, I think that was because we of our `fake_machine_token` fixture. We need to import that file in way that the mock in that fixture will always work on every module. That's why I have changed the imports to follow this pattern

Reply:
Yeah, I agree we can go with `from uaclient.files import machine_token`